services/events/events_definitions/models.go

```diff
@@ -0,0 +1,31 @@
+package events_definitions
+
+import "github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
+
+type EventsDefinition struct {
+	ID             string       `json:"id"`
+	Name           string       `json:"name"`
+	Source         string       `json:"source"`
+	State          string       `json:"state"`
+	Description    string       `json:"description,omitempty"`
+	Mappings       []Mappings   `json:"mappings,omitempty"`
+	Rules          []Rules[any] `json:"rules,omitempty"`
+	CreatedAt      string       `json:"createdAt"`
+	CreatedBy      string       `json:"createdBy"`
+	LastModifiedAt string       `json:"lastModifiedAt"`
+	LastModifiedBy string       `json:"lastModifiedBy"`
+}
+
+func (eventDefinition *EventsDefinition) GetID() string { return eventDefinition.ID }
+
+type Rules[T any] struct {
+	Operator       string                             `json:"operator"`
+	Path           string                             `json:"path"`
+	ValueAttribute *general_objects.ValueAttribute[T] `json:"valueAttribute"`
```

Comment:
it's weird that it is called valueattribute, right? (not here but in the API)

Reply:
It's on API :/ 
You can ask on Gold space to modify it

Reply:
Ticket to rename the field: https://leanspace.atlassian.net/browse/GOLDS-797. It's in progress.

Reply:
The PRs of ticket https://leanspace.atlassian.net/browse/GOLDS-797 were merged and deployed on develop. You can apply the change/renaming of the ValueAttribute field here.